.ci/containers/downstream-builder/generate_downstream.sh

```diff
@@ -84,6 +84,7 @@ fi
 
 if [ "$REPO" == "terraform" ]; then
     pushd $LOCAL_PATH
+    go mod download
```

Comment:
Seems like this was after `find` before, but it's before it now. Any specific reason for the change?

Reply:
Yeah, we need this since when we changed gomod to be managed upstream we deleted it in the 
```find . -exec rm```

so when the container was updated nothing was being downloaded in the go mod download.

https://github.com/GoogleCloudPlatform/magic-modules/commit/f23e23890521c0398b3845b4dfce66caaffa89b1#diff-44d9686dbaca364b4030b60fc69710e6dc207ed12f026d530ed48eeb8577c82eR87